digital-islandupdate.sh

```diff
@@ -104,6 +104,9 @@ mv ./pdf.js ./pdfjs
 rm -R jstree
 git clone https://github.com/vakata/jstree.git
 
+# JAIL library
+rm -R JAIL
```

Comment:
@pc37utn, ignore this comment.  for some reason was thinking you wanted to drop JAIL, but then remembered that was JW Player.